src/esp/gfx/PbrShader.cpp

```diff
@@ -362,6 +405,8 @@ PbrShader& PbrShader::setTextureMatrix(const Mn::Matrix3& matrix) {
                  "PbrShader::setTextureMatrix(): the shader was not "
                  "created with texture transformation enabled",
                  *this);
+
+  // since emissive texture may need it, so no if (lightCount_) here
   setUniform(textureMatrixUniform_, matrix);
```

Comment:
This may still result in warnings from GL, so maybe 

```suggestion
  if (flags_ & (Flag::TextureMatrix|Flag::EmissiveTexture))
    setUniform(textureMatrixUniform_, matrix);
```